### ISSUE TITLE
LT using only 1 porocessor when tagging.

### ISSUE THREAD

**Issue by baarsrj** [languagetool-org/languagetool#10038]
Is there a specific reason why LT does only use 1 processor when tagging?

java -Dfile.encoding=utf-8 -jar languagetool-commandline.jar -t -b -l nl -c "UTF-8" -r $CORPUS > $LTROOT/tagged.txt

**Comment by danielnaber** [comment]
> Is there a specific reason why LT does only use 1 processor when tagging?

No, it's just such a specific use case that nobody ever thought about optimizing this, I think.

**Comment by baarsrj** [comment]
That's a pity. Tagging the entire data set will be taking 3 full days now, instead of just one. 
Strange, by the way that is is hardly used. It is the best way to get an idea of how the assigning of postags is working, by getting the patterns and analysing them.
